accounts/models.py

```diff
@@ -55,6 +55,20 @@ class UserProfile(AbstractBaseUser, TimeStampedModel, PermissionsMixin):
             "Unselect this instead of deleting accounts."
         ),
     )
+    first_name = models.CharField(max_length=50, blank=False)
```

Comment:
@autn for models, `blank=False` is the default so this isn't technically necessary.